src/QMCWaveFunctions/ElectronGas/FreeOrbitalBuilder.cpp

```diff
@@ -31,6 +34,14 @@ std::unique_ptr<SPOSet> FreeOrbitalBuilder::createSPOSetFromXML(xmlNodePtr cur)
   app_log() << "twist cartesian = " << tvec << std::endl;
 #else
   npw = std::ceil((norb + 1.0) / 2);
```

Comment:
Remove int npw in line 15 and put
`int npw =` here and line 31
Also please do a clang-formatting.